src/main/webapp/app/markdown-editor/markdown-editor.component.ts

```diff
@@ -76,6 +77,8 @@ export class MarkdownEditorComponent implements AfterViewInit {
     /** {string} which is initially displayed in the editor generated and passed on from the parent component*/
     @Input() markdown: string;
     @Input() editorMode = EditorMode.NONE;
+    // If false will hide the line numbers.
+    @Input() showGutter = false;
```

Comment:
can we rename this input to something like showLineNumbers?

Reply:
You are right, then we also don't need a comment for it.